models/Contact.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+const Language = require('./Language').schema;
+const ImageSet = require('../modules/common/ImageSet').schema;
+
+const ContactSchema = new mongoose.Schema({
+  phoneNumber: Number,
+  openHours: [Language],
+  address: {
+    city: [Language],
+    street: [Language],
+    buildingNumber: [Language],
```

Comment:
I think it is a good idea to add `postalCode` too for returning the products

Reply:
Done